src/main/java/reposense/util/FileUtil.java

```diff
@@ -38,6 +41,52 @@
     private static final String GITHUB_API_DATE_FORMAT = "yyyy-MM-dd";
     private static final ByteBuffer buffer = ByteBuffer.allocate(1 << 11); // 2KB
 
+    /**
+     * Returns a {@code set} of relevant repo paths from {@code repoConfigsList}.
```

Comment:
`set` dont have to highlighted with `{@code ...}` 
Also, the comment needs to improve. How do you determine `relevant`?